data/hai/hai reveal 1 intro.txt

```diff
@@ -0,0 +1,947 @@
+	
+ship "Geodesic Geocoris" "Geodesic Geocoris (Phylactery)"
+	outfits
+		"Electron Turret" 3
+		"Chameleon Anti-Missile" 2
+		"AR120 Reverse Thruster" 2
+		"Boulder Reactor"
+		"Hai Ravine Batteries"
+		"Outfits Expansion"
+		"Large Heat Shunt"
+		"Quantum Keystone"
+		"Pulse Rifle" 14
+		`"Benga" Atomic Thruster`
+		`"Bufaer" Atomic Steering`
+		"Jump Drive"
+		Hyperdrive
+		"Failed Control System"
+	turret "Electron Turret"
+	turret "Electron Turret"
+	turret "Chameleon Anti-Missile"
+	turret "Chameleon Anti-Missile"
+	turret "Electron Turret"
+
+mission "Hai Leaks Response 5"
+	landing
+	name "Talk to Hai authorities to <planet>"
+	description "Meet Hai authorities on <destination>."
+	source "Allhome"
+	clearance
+	destination "Hai-home"
+	to offer
+		has "Hai Leaks Response 4: done"
+
+mission "Hai Leaks Response 6"
+	name "Investigate rumours of kidnapped Hai."
+	description "It's a statistical oddity, but Sayari's friend thinks someone is kidnapping Hai. Check spaceports in nearby parts of Hai and human space for evidence of a Hai trafficking operation."
+	autosave
+	landing
+	source "Hai-home"
+	clearance
+	to offer
+		has "Hai Leaks Response 4: done"
+	on offer
+		event "secret mountaintop"
+		event "role-play fleet deception"
+		# Hai like you a lot now, and as a result, you have permanent access to Hai-home:
+		"reputation: Hai" >?= 100
+		log `The working solution to the secrecy crisis for now is to have the Hai post a number of ships on the other side of the wormhole to play-act as an isolated furry role-play community. This is only a short-term solution. While they are working on a longer-term solution Sayari has quietly asked you to look into a rumour about more kidnapped Hai. She does not consider it to be a particularly likely scenario, but her contacts insist and so she would like you to be sure there's no issues lurking while the diplomatic process is working.`
+		conversation
+			`When you return to Hai-home you are met by a group of relieved diplomats.`
+			`	Of all of them Alondo seems to be the least tired and takes the time to let you know what's been going on, "Captain <last>! You've missed quite the palava. Still, I doubt you lost much by missing out on trying to explain to Sayari what a 'furry' is."`
+			choice
+				`	"Oh I don't know, could've been entertaining."`
+				`	"Hmm, yes, no probably not."`
+			`	He chuckles, "Anyway, I have a hunch that we're going to be needing you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun."`
```

Comment:
```suggestion
			`	He chuckles. "Anyway, I have a hunch that we'll need you before too long, but for now they're putting some Hai patrols out on the other side of the wormhole with a couple of Navy observers on board to 'play the part', as it were, of an exclusive role-play community, just chilling out here and having fun. Basically, we're going to pretend the Hai are furries for now."`
```

Reply:
Did we reeeaaally have to say it out loud?

Reply:
Understandable, have a great day (suggestion updated)